packages/styles/components/Button.module.scss

```diff
@@ -235,6 +245,12 @@
   }
 
   &.medium {
+    padding-left: var(--space-sm);
+    padding-right: var(--space-sm);
+    width: 40px;
```

Comment:
I think that you wanted to said `height` 😂

Reply:
oh god hahahahahahahahahaha my fault, sorry

Reply:
not really. I'm just follow the style from other sizes, since the width is already setted in the `medium` class.

```css
.circle {
  border-radius: 50%;

  &.small {
    width: var(--space-md);
  }

  &.medium {
    padding-left: var(--space-sm);
    padding-right: var(--space-sm);
    width: 40px;
  }

  &.large {
    width: var(--space-lg);
  }
}
```

Reply:
Ohhh yeah, Github confused me because file was collapsed, hahaha 😂 😂 😂 😂